services/workflows-service/prisma/schema.prisma

```diff
@@ -833,18 +844,21 @@ model BusinessReport {
   reportId String
   report Json
 
+  riskScore  Int
   businessId String
+  business   Business @relation(fields: [businessId], references: [id])
+
   projectId  String
```

Comment:
projectId field seems to be redundant here, you already have this association on each business